extensions/amp-sidebar/0.1/test/test-amp-sidebar.js

```diff
@@ -379,27 +399,27 @@ describes.realWin(
           owners.schedulePause = sandbox.spy();
           owners.scheduleResume = sandbox.spy();
 
-          expect(impl.isOpen_()).to.be.false;
+          expect(sidebarElement.hasAttribute('open')).to.be.false;
```

Comment:
Do these tests still test what they're supposed to test now that we don't use this attribute to guard whether or not the sidebar is `open`?

Reply:
I think they still check the attribute is set, which can be used for styling.